src/app/frontend/replicationcontrollerdetail/serviceendpoint_directive.js

```diff
@@ -22,6 +22,7 @@ export default function serviceEndpointDirective() {
     templateUrl: 'replicationcontrollerdetail/serviceendpoint.html',
     scope: {
       'endpoint': '=',
+      'url': '@',
```

Comment:
@floreks I think this is not feasible. We would have to ping the server. This would be slow and might do harm to it.